pkg/lib/web/static_asset_resolver.go

```diff
@@ -149,13 +122,8 @@ func ParsePathWithHash(hashedPath string) (filePath string, hash string) {
 	return
 }
 
-func IsAssetPathHashed(assetPath string) bool {
-	id := strings.TrimPrefix(assetPath, StaticAssetResourcePrefix)
-	desc, ok := StaticAssetResources[id]
-	if !ok {
-		return false
-	}
-	return StaticAssetResourcesIsHashed[desc]
+func IsAssetPathHashed(hash string) bool {
```

Comment:
Do not alter the meaning of an existing function. Delete the original function and create a new one with a name telling what it does. Here, `LookLikeAHash(s string) bool` would be a good name.